types/api/tooltip.d.ts

```diff
@@ -1,7 +1,8 @@
 /**
  * Tooltip API
  */
-import {TooltipContent, TooltipOptions} from 'codex-tooltip';
+import TooltipOptions from 'codex-tooltip';
+import TooltipContent from 'codex-tooltip';
```

Comment:
Looks strange. Please, describe what you've discovered

Reply:
![codex-tooltip](https://user-images.githubusercontent.com/23116557/150162660-b7b954de-9e38-449b-bcc7-019e75e46918.png)

`TooltipOptions` and `TooltipContent` are only imported in `node_modules/codex-tooltip/dist/src/index`.
They need to be exported too to be accessible?

Reply:
fix angular build. PR-1966 is a clone also